script.js

```diff
@@ -0,0 +1,15 @@
+const count0 = document.querySelector('#count--0');
+const resetbutton = document.getElementsByClassName('btn--new');
+const clickbutton = document.querySelector('.btn--click');
+let count=0;
+function addcount(){
+    document.querySelector('#count--0').textContent=count++;
```

Comment:
**BUG**
because of this code **count++ this is post fix** operator which will increment value later not at time You should use this **++count prefix operator** which will remove your bug and read about this operator so you will get clear understanding